Code/Framework/AzCore/AzCore/RTTI/TypeInfoSimple.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Math/Uuid.h>
+#include <AzCore/std/typetraits/is_enum.h>
+
+namespace AZ
+{
+    using TypeId = AZ::Uuid;
+}
+
+#define AZ_TYPE_INFO_INTERNAL_1(_ClassName) static_assert(false, "You must provide a ClassName,ClassUUID")
+#define AZ_TYPE_INFO_INTERNAL_2(_ClassName, _ClassUuid)        \
+    void TYPEINFO_Enable(){}                                   \
+    static const char* TYPEINFO_Name() { return #_ClassName; } \
+    static const AZ::TypeId& TYPEINFO_Uuid() { static AZ::TypeId s_uuid(_ClassUuid); return s_uuid; }
```

Comment:
It looks like the `AZ_TYPE_INFO_INTERNAL_1` and `AZ_TYPE_INFO_INTERNAL_2` can be removed from the TypeInfo.h header above.

Furthermore they deprecated macros for AZ_TYPE_INFO_X which takes more than 2 parameters can be removed since no structs/classes should be using them at this point in time.

Reply:
Will do, thank You